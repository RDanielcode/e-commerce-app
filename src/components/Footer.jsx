import React from 'react';
import '../styles/components/Footer.css'

const Footer = () => {
  return (
    <div className="Footer class">
      <p className="Footer-title">Store</p>
      <p className="Footer-copy">All rights reserved</p>
    </div>
  );
}

export default Footer;