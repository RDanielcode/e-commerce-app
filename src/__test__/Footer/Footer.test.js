import React from 'react'
import Footer from '../../components/Footer'
import { create } from 'react-test-renderer'
import { mount } from 'enzyme'

describe('<Footer />', ()=> {
  const footer = mount(<Footer />)
  test('Footer render', () => {
    expect(footer.length).toEqual(1)
  })
  test('footer title', ()=>{
    expect(footer.find('.Footer-title').text()).toEqual('Store')
  })
})

describe('Footer snapshot', ()=>{
  test('testing ui', ()=>{
    const footer = create(<Footer />)
    expect(footer.toJSON()).toMatchSnapshot()
  })
})