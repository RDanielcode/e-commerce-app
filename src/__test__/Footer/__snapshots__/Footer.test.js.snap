// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot testing ui 1`] = `
<div
  className="Footer class"
>
  <p
    className="Footer-title"
  >
    Store
  </p>
  <p
    className="Footer-copy"
  >
    All rights reserved
  </p>
</div>
`;
